content/php/concepts/string-functions/terms/html-entity-decode/html-entity-decode.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'PHP html_entity_decode() Function'
+Description: 'Converts HTML entities to characters.'
+Subjects: 
+- 'Web Design'
+- 'Web Development'
+Tags:
+- 'Functions'
+- 'Characters'
+CatalogContent:
+- 'php/concepts/string-functions/terms/html-entity-decode'
+
+## About this function
+
+html_entity_decode() is the opposite of htmlentities() in that it converts HTML entities in the string to their corresponding characters.
+
+## SyntaxSection
+```pseudo
+html_entity_decode(str, flags, character-set)
+
+##Example:
+```php
+<?php
+$orig = "dog";
+$a = htmlentities($orig);
+$b = html_entity_decode($a);
+echo $a;
+echo "\n";
+echo $b;
+?>
```

Comment:
add the closing tag after this please like ```

Reply:
I see it in my version below the line: "?>"?